### ISSUE TITLE
Problem with migration to 1.0, `indent` rule.

### ISSUE THREAD

**Issue by zloirock** [eslint/eslint#3228]
Old rule: `"indent": [2, 2]`.
New rule: `"indent": [2, 2, {"VariableDeclarator": 2, "SwitchCase": 1}]`.
[Example 1](https://github.com/zloirock/core-js/blob/v1.0.1/modules/%24.array-methods.js#L27):
```js
module.exports = function(TYPE){
  // ..
  return function($this, callbackfn, that){
    var O = toObject($this, true)
      // ...
      , val, res;
    // 27:34 error Expected indentation of 6 characters but found 4 indent
    for(;length > index; index++)if(NO_HOLES || index in self){
```
[Example 2](https://github.com/zloirock/core-js/blob/v1.0.1/modules/%24.ctx.js#L7):
```js
module.exports = function(fn, that, length){
  aFunction(fn);
  if(~length && that === undefined)return fn;
  switch(length){
    // 7:13 error Expected indentation of 6 characters but found 4 indent
    case 1: return function(a){
```

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by gyandeeps** [comment]
Working on this.

**Comment by targos** [comment]
Cross-referencing: https://github.com/nodejs/io.js/pull/2286
